elide-datastore/elide-datastore-hibernate/src/main/java/com/yahoo/elide/core/hibernate/hql/SubCollectionFetchQueryBuilder.java

```diff
@@ -24,10 +26,11 @@
 
     private final Relationship relationship;
 
-    public SubCollectionFetchQueryBuilder(Relationship relationship,
+    public SubCollectionFetchQueryBuilder(EntityProjection entityProjection,
+                                          Relationship relationship,
```

Comment:
This method should get the entityProjection from the relationship - so there is no parameter duplication